docs/troubleshooting.md

```diff
@@ -0,0 +1,38 @@
+# Troubleshooting
+
+## PR-traffic-light failures:
+#### Operator changes detected with ci changes and 'allow/ci-changes' is not set
+You are unintentionally editing a file which is out of your responsibility, please rebase.
```

Comment:
I would write here. to "Do not modify files outside of community-operators and upstream-community-operators". Please fix it and push changes

Reply:
Thanks for great ideas. I will add. Just to explain why rebase was proposed. We already had a case where contributor was not editing our pipeline, just he was not rebased.